django_resized/tests/models.py

```diff
@@ -13,4 +13,8 @@ class Product(models.Model):
     image5 = ResizedImageField(size=[500, 350], upload_to=UPLOAD_TO, blank=True, quality=10)
     image6 = ResizedImageField(size=[500, 350], upload_to=UPLOAD_TO, keep_meta=False, blank=True, quality=10)
     image7 = ResizedImageField(size=[2000, 2000], upload_to=UPLOAD_TO, blank=True)
+    image8 = ResizedImageField(size=[100, 100], crop=['top', 'right'], scale=2.0, upload_to=UPLOAD_TO, blank=True)
+    image9 = ResizedImageField(size=[None, None], scale=0.5, upload_to=UPLOAD_TO, blank=True)
```

Comment:
I could do that, but since size has always been of list type, wouldn't it be interesting to remain this way? There is also the convention that the dimension with 'None' is the one which has not been set:

1920x1080 -> size=[192, *None*] -> scale=0.1 -> 192x*108*
500x1000 -> size=[*None*, 500] -> scale=0.5 -> *250*x500
640x640 -> size=[*None*, *None*] -> scale=2.0 -> *1280*x*1280*

Reply:
I think there is a difference between having only a single dimension in the size or not having a size at all.

Maybe we should support both.

Reply:
👍👍